assets/js/load-chart.js

```diff
@@ -166,7 +169,8 @@ function setupChart () {
                 'beforeend',
                 `<tr class="${idx % 2 ? 'odd' : 'even'}">
                     <td>${scores.focus_areas[area]}</td>
-                    <td class="score">${String(recent_score[idx + AREA_SCORE_OFFSET] / 10).padEnd(4, '.0')}%</td>
+                    <td class="score">${String(recent_score[idx + AREA_SCORE_OFFSET].per_mille / 10).padEnd(4, '.0')}</td>
+                    <td class="score">(${String(recent_score[idx + AREA_SCORE_OFFSET].total_subtests_passed)}/${String(recent_score[idx + AREA_SCORE_OFFSET].total_subtests)}) ${String(recent_score[idx + AREA_SCORE_OFFSET].per_mille_subtests / 10).padEnd(4, '.0')}% </td>
```

Comment:
This is quite a long expression. Can we extract some of the subexpressions into variables to make this more readable?

Reply:
Fixed on the newer version.